include/kuzzlesdk.h

```diff
@@ -132,6 +133,8 @@ typedef struct {
     const char *limit;
     unsigned long count;
     const char *match;
+    const bool reset;
```

Comment:
If you are using a `const bool` here (which I approve), why didn't use `const double` `const size_t` `const long` also ?